examples/meta/src/distance/manhattan_distance.sg

```diff
@@ -0,0 +1,18 @@
+CSVFile f_feats_train("../../data/fm_train_real.dat")
```

Comment:
not sure "train" and "test" is the best name here. You never train or test with a model.
We want to illustrate two things: Evaluating a symmetric distance, and one between different data.
Therefore, just call them features_a, features_b maybe